cirq/experiments/xeb_fitting.py

```diff
@@ -453,22 +459,36 @@ def _fit_exponential_decay(cycle_depths: np.ndarray, fidelities: np.ndarray) ->
         a: The first fit parameter that scales the exponential function, perhaps accounting for
             state prep and measurement (SPAM) error.
         layer_fid: The second fit parameters which serves as the base of the exponential.
+        a_std: The standard deviation of the `a` parameter estimate.
+        layer_fid_std: The standard deviation of the `layer_fid` parameter estimate.
     """
     cycle_depths = np.asarray(cycle_depths)
     fidelities = np.asarray(fidelities)
 
     # Get initial guess by linear least squares with logarithm of model
     positives = fidelities > 0
+    if np.sum(positives) <= 1:
+        return 0, 0, np.inf, np.inf
```

Comment:
Could you comment on this here?

Reply:
yes; this is a bit of numpy trickery. Added comment(s)